README.md

```diff
@@ -13,7 +13,19 @@ Git during building your application please make sure to include the Git history
 
 ## Evaluation
 
-You should turn in a complete and runnable solution, but do not gold-plate it: we don't expect you to spend more than a few hours on this exercise. We want to see that you know how to use APIs efficiently, how you structure and how you are testing your application. It should be easy to tell from the dashboard which ministries have provided the most data.
+We want to use this challenge to understand how you would approach such a problem in a work setting. Here are some specifics of what we are looking for:
+
+Among other things, we pay attention to:
+- how you structure your application and your source code,
+- how you name variables/functions/classes/packages,
+- how you test your application and what kinds of tests you're writing,
+- how you document your application, including instructions to run the project,
+- how you write your git commit messages,
```

Comment:
Further up in the text it sounds like using git is optional though. Maybe make it mandatory (I'd vote for that) or also add "if you use git" here?

Reply:
Let's generalize this some more - it's not just about the commit message, but whether a developer commits atomic changesets or various unrelated changes in a single huge commit. There's some correlation here: atomic commits will allow you to write much better commit messages, as you need to describe only one thing at a time..

Reply:
So, something like: "How your commit log looks like" ... 🤔

Reply:
Gotcha. I've made VCS mandatory (doesn't have to be git, I think?) and have generalized the expectation to "what your commits look like". Does that sit right with you @benjaminaaron @carhartl ?